libtransmission/bitfield.cc

```diff
@@ -157,37 +154,35 @@ size_t Bitfield::countBits() const
     return this->true_count_;
 }
 
-void Bitfield::setBitsInArray(uint8_t* array, size_t bit_count)
+void Bitfield::setBitsInArray(std::vector<uint8_t> &array, size_t bit_count)
 {
-    uint8_t const val = 0xFF;
-    size_t const n = getStorageSize(bit_count);
+    size_t const byte_count = getStorageSize(bit_count);
 
-    if (n > 0)
+    if (byte_count > 0)
     {
-        memset(array, val, n - 1);
+        std::fill(array.begin(), array.begin() + byte_count - 1,  0xFF);
```

Comment:
- prefer `std::begin(foo)` over `foo.begin()` here and elsewhere
- this would be a good candidate for `std::fill_n()` to avoid the second begin() call

Reply:
Changed begin, end and empty to std:: variants. Used fill_n instead of fill.